protos/foundation/protos/doc/entity.proto

```diff
@@ -87,13 +97,15 @@ message Table {
 	 words into multiple OCR rectangles.
  */
 message Token {
```

Comment:
How do we think about Token? I think it might be weird that it's span and box are just duplicate information?

Reply:
I would get rid of token until a real need arises and we can clearly differentiate it from these other types (both conceptually and technically).

Reply:
I agree, I think Word is sufficient?

Reply:
sounds good to me